code/game/objects/items/explosives/bombvest.dm

```diff
@@ -0,0 +1,18 @@
+/obj/item/clothing/suit/storage/marine/harness/boomvest
+	name = "\improper Tactical Explosive Vest"
+	desc = "Obviously someone just strapped a bomb to a marine harness and called it tactical. The light has been removed, and its switch used as the detonator. <span class='notice'> Control-Click to set a warcry.</span> <span class='warning'>This harness has no light, toggling it will detonate the vest!</span>"
```

Comment:
```suggestion
	desc = "Obviously someone just strapped a bomb to a marine harness and called it tactical. The light has been removed, and its switch used as the detonator.<br><span class='notice'>Control-Click to set a warcry.</span> <span class='warning'>This harness has no light, toggling it will detonate the vest!</span>"
```